pjmedia/src/pjmedia/vid_conf.c

```diff
@@ -771,10 +786,22 @@ static void on_clock_tick(const pj_timestamp *now, void *user_data)
 	    frame.size = sink->put_buf_size;
 	}
 	status = pjmedia_port_put_frame(sink->port, &frame);
-	if (status != PJ_SUCCESS) {
-	    PJ_PERROR(5, (THIS_FILE, status,
-			  "Failed to put frame to port [%s]!",
-			  sink->port->info.name.ptr));
+	if (got_frame && status != PJ_SUCCESS) {
+	    if (vid_conf->last_err[i] != status ||
+	        vid_conf->last_err_cnt[i] > MAX_ERR_COUNT ||
+	        status != PJ_EINVALIDOP)
```

Comment:
- exclude this condition 'status != PJ_EINVALIDOP'?
- also print the error counter

Reply:
what about printing the error counter (if it is more than 1)? should be helpful to identify 'persistent' errors?

Reply:
currently the error counter will always be reset.
unless we change it to:
`sink->last_err_cnt % MAX_ERR_COUNT == 0`

there are a couple of drawbacks:
- require division
- can overflow

or perhaps we don't need the actual value of the counter itself. then we can simply print:
`failed again to ....`

Reply:
For now, we need to identify any persistent/recurring error, so being reset or overflow is not really a problem I guess, e.g: 50 same errors should already present it.

Using 'again' sounds like just showing a second occurrence (after few minutes), it is fine as long as we can remember what it really means (like next month :), as generally I prefer to remember less things, I'd vote for simple logging the counter number as is, IMO any not small number (e.g: > 10) will do.